packages/create-next-app/templates/experimental-app/app/layout.tsx

```diff
@@ -1,8 +1,16 @@
-export default function RootLayout({ children }) {
+import '../styles/globals.css'
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
   return (
     <html lang="en">
       <head>
         <title>Create Next App</title>
+        <meta name="description" content="Generated by create next app" />
```

Comment:
```suggestion
        <meta name="description" content="Generated by create-next-app" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Reply:
`<meta name="viewport" content="width=device-width, initial-scale=1" />` will be auto-added so shouldn't be placed here and instead would go in `head.js`

Reply:
Ack, sounds good